frontend/src/styles/register.scss

```diff
@@ -0,0 +1,76 @@
+.register {
+	display: flex;
+	justify-content: center;
+
+	.regContainer {
+		width: 100%;
+		max-width: 700px;
+
+		.form {
+			display: flex;
+			justify-content: space-evenly;
+
+			@media screen and (max-width: 1400px) {
+				flex-direction: column;
+			}
+
+			form {
```

Comment:
Your registration form seem to be a single column for both mobile and desktop view, instead of 2-columns on desktop. 
Kindly confirm from the figma provided.

Reply:
It's two columns on desktop from the figma